modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -135,16 +135,38 @@ When you update the agent, the `amc_setup` script:
 +
 This folder contains all custom modules added to the agent (not included in the agent distribution).
 
+== Roll Back the Agent
+
+If you need to roll back the agent, for example, if problems occur when you update the agent,
+use the `mule-agent-backup.zip` file to restore the previous version.
+The `mule-agent-backup.zip` file includes libraries, configuration files, and compiled classes for the agent.
+
+To roll back the agent:
+
+. Stop Mule or the API gateway runtime.
+. Run the following commands to restore the previous version of the agent from the `mule-agent-backup.zip` file:
++
+[source,console,linenums]
+----
+$ cd $MULE_HOME/server-plugins/mule-agent-plugin
+$ cp $MULE_HOME/tools/mule-agent-backup.zip .
```

Comment:
// Delete all files in the mule-agent-plugin folder
// Warning: please double check you are in the $MULE_HOME/server-plugins/mule-agent-plugin folder
$ rm -r *